src/cloud-api-adaptor/podvm-mkosi/Makefile

```diff
@@ -76,6 +76,7 @@ else
 	touch resources/buildBootableImage
 	nix develop ..#podvm-mkosi --command mkosi --environment=VARIANT_ID=production
 	qemu-img convert -f raw -O qcow2 build/system.raw build/podvm-$(PODVM_DISTRO)-$(ARCH).qcow2
+	qemu-img resize build/podvm-$(PODVM_DISTRO)-$(ARCH).qcow2 +100M
```

Comment:
Is there any significance to +100M, or is it just a starter amount of space?

Reply:
it needs to have a bit of size, so repart will not stumble over it at launch, attempting to create a partition without space. not sure if needs to be +100m, could just also work with +10m, have to test.

Reply:
Ok, I don't have an issue with the 100M, just wanted to check if there was special logic behind it.

Reply:
This is only being done on the x86 path here as only Azure is supporting this scratch space at the moment I assume? Do you see any obstacles in the code for s390x support if/when needed?

Reply:
I don't know much about partitioning schemes on s390x. if systemd-repart works in the same way, it should work ootb. The cloud-provider code would need to consider the `.storage` property when provisioning a VM and then repart would grab whatever it available to allocate it to the scratch space.